bevy-visualizer/src/lib.rs

```diff
@@ -15,32 +15,6 @@ struct Led(usize);
 struct Points(Vec<(f32, f32, f32)>);
 impl Resource for Points {}
 
-/// set up a simple 3D scene
-pub fn create_plane_and_light(
```

Comment:
I wonder if we should have some way of configuring between the emissive lights and the previous version, as it was giving a bit more perspective in cases where we could afford it.
What do you think about adding a feature for it?

Reply:
I'd rather spend some time figuring out how to make the new version look better (including perspective), instead of keeping and maintaining two versions.

Reply:
Fair enough